### PR TITLE
Improve help message for create option

### LINKED ISSUE #28: Improve help message when using "plonecli create"
* Plone CLI version:

0.2.0

* Python version:

2.7.9

* Operating System:

### Description

Improve help.

### What I Did

```
plonecli create                                                                                                                                                                 [17:14:23]
Usage: plonecli create [OPTIONS] TEMPLATE NAME
Try "plonecli create -h" for help.

Error: Missing argument "template".
```

And then:

```
plonecli create -h                                                                                                                                                              [17:14:27]
Usage: plonecli create [OPTIONS] TEMPLATE NAME

  Create a new Plone package

Options:
  -h, --help  Show this message and exit.
```

The first message `Try "plonecli create -h" for help.` is useless in this context. `plonecli -l` is much more useful in this case.

### PR DESCRIPTION
**Pull request by mzfr**
Fixes #28 
I have simply changed the message for `create` option.
Please let me know any other changes are required.

### CODE REVIEW

**Comment by mzfr** [comment]
@MrTango @kakshay21 Do I have to mention every change in `CHANGE.rst` file?

File: plonecli/cli.py [hunk 1]
```diff
@@ -61,7 +61,9 @@ def cli(context, list_templates, versions):
     @click.argument('name')
     @click.pass_context
     def create(context, template, name):
-        """Create a new Plone package"""
+        """Create a new Plone package.
+        Run "Plonecli -l" to see available templates
```
Comment (kakshay21) [inline plonecli/cli.py]: Use `\n` for new line.

**Comment by kakshay21** [comment]
@mzfr  yeah, you should add a line explaining about this PR in `CHANGES.rst`.

File: plonecli/cli.py (hunk 1 above)
Comment (mzfr) [reply plonecli/cli.py]: @kakshay21 I have shifted that message to a new line.

**Comment by idgserpro** [comment]
> At this point the user already know how to use the plonecli, probably by using the intr help command

We don't agree with this assumption. This issue was raised by people not familiar with plonecli in the first place.

Anyways. If you really think this is useless, you can close the related issues in:

https://github.com/plone/plonecli/issues/28
https://github.com/plone/plonecli/issues/29

**Pull request closed without merging**